development/prow-installer/cmd/installer/main.go

```diff
@@ -31,8 +33,13 @@ func main() {
 
 	readConfig, err := config.ReadConfig(*configPath)
 	if err != nil {
-		log.Fatalf("Error reading configPath file %v", err)
+		log.Fatalf("Error reading config file %v", err)
 	}
+	// define mandatory environment variables
+	_ = os.Setenv("GOOGLE_APPLICATION_CREDENTIALS", *credentialsFile)
+
+	kubeHelpers := make(map[string]*kubehelper.KubeHelper) // map of k8s cluster helpers to communicate with them
```

Comment:
why do we need a map here, when it only holds one cluster in the code below?

Reply:
It was made with the accessing multiple clusters in mind like prow and workload ones. When creating specific cluster (prow, workload) we don't know which cluster is for what. I came up with the idea that it should be easier with defining a map of clients, than looking for another solution of accessing multiple clusters.